packages/design-system-scripts/gulp/build.js

```diff
@@ -37,7 +39,17 @@ function copySass(dir) {
 }
 
 function copyAll(dir) {
-  return Promise.all([copyAssets(dir), copyJson(dir), copySass(dir)]);
+  // Check to see if this is a core package. If it's not, copy assets from core npm package
+  const copyAssetsTask = async (dir) => {
```

Comment:
@pwolfert I moved the copying from core npm package logic here, in order to generalize/simplify `copyAssets`. This way after I remove the temp dir, I can use it [here](https://github.com/CMSgov/design-system/pull/659/commits/af42b932eb28752c0f22ffa2e6d7155e51a31723#diff-4cbc3eb22061dce437a86ae65943d8d6R53) in the build-docs script. I'm still wrapping my head around some of this, so lmk if this looks about right to you

Reply:
Yeah, that works too. I had only intended using this for the source package and not the docs package since the docs package only has images and not fonts. This anonymous function below, however, will _either_ copy from core _or_ the `dir`. I believe you want it to `await` both of the `copyAssets` calls.

Reply:
Should be fixed here: https://github.com/CMSgov/design-system/pull/659/commits/6c6747329cd09fc51bfea78f3108865df4ec5faf